flyway/sql/V1.1__createSampleTable.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE sample (
```

Comment:
Rename the file to not have `V1.1` in the name. Instead, go with a 3 character sequence like `001`. so that incrementing from 009 --> 010 will still have them ordered correctly in the OS and your text editor.

Reply:
V001 or just 001?